index.html

```diff
@@ -40,9 +40,9 @@ <h3 id="pricing1">Feature</h3>
   
           <iframe id="video" height="315" src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0"></iframe>
 
```

Comment:
It will be **# invaluable** to you if you made it a habit to ***document your code*** as Any Future rework and references to your codebase will have been achieved by *halfway*.